emails/2024-02-09.md

```diff
@@ -0,0 +1,27 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+We bought some Dolt-branded hoodies. I wore it home and had the following exchange with my 10-year old son:
+
+Son: "You're selling merch now? I want one."
+Me: "Dolt means idiot."
+Son: "I'm good."
+
+### Test Containers Works With Dolt
+
+We're on a mission to show that Dolt just works with all your favorite tools in all your favorite languages. This week, [the spotlight](https://www.dolthub.com/blog/2024-02-05-testcontainers-for-go-with-dolt/) is on [Test Containers](https://testcontainers.com/), a framework for testing using Docker containers. 
+
+Dolt is useful in test because you can clone data from DoltHub, run write tests, and reset after every test. We [use this approach to test Dolt itself](https://www.dolthub.com/blog/2022-06-10-enginetest-perf/) and sped up our tests 10X. If you're looking for a non-production Dolt use case and have a MySQL database, consider Dolt as your test database [using Test containers](https://www.dolthub.com/blog/2024-02-05-testcontainers-for-go-with-dolt/).
```

Comment:
```suggestion
Dolt is useful in test because you can clone data from DoltHub, run write tests, and reset after every test. We [use this approach to test Dolt itself](https://www.dolthub.com/blog/2022-06-10-enginetest-perf/) and sped up our tests 10X. If you're looking for a non-production Dolt use case and have a MySQL database, consider Dolt as your test database [using Test Containers](https://www.dolthub.com/blog/2024-02-05-testcontainers-for-go-with-dolt/).
```